lib/environment.rb

```diff
@@ -415,7 +415,7 @@ def nodes(user: admin, refresh: false)
     def service_project
       unless @service_project
         project = Project.new(name: "tests-" + EXECUTOR_NAME.downcase, env: self)
-        unless project.exists?
+        unless project.exists? && project.active?
```

Comment:
For better performance I'd suggest using `project.active?(cached: true)`.

Reply:
okay I'll update and try to figure out a good was to test triggering the `Terminating` phase.